recipes-devtools/python/python3-labgrid/labgrid-coordinator.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Labgrid Coordinator
+After=network.target
+
+[Service]
+Environment="PYTHONUNBUFFERED=1"
+Environment="LABGRID_COORDINATOR_LISTEN_IP=0.0.0.0"
+Environment="LABGRID_COORDINATOR_PORT=20408"
+EnvironmentFile=/etc/labgrid/environment
+ExecStart=/usr/bin/labgrid-coordinator -l ${LABGRID_COORDINATOR_LISTEN_IP}:${LABGRID_COORDINATOR_PORT}
+Restart=on-failure
+DynamicUser=yes
+StateDirectory=labgrid-coordinator
+# Set WorkingDirectory to StateDirectory, this works in DynamicUser mode since symlinks are created
```

Comment:
If we drop `DynamicUser=yes`, the comment won't apply anymore.

Reply:
Copied from https://github.com/labgrid-project/labgrid/blob/master/contrib/systemd/labgrid-coordinator.service
Happy to change to a better suited version for yocto. But we should first clarify if we want a separate package or not.